server/src/db.js

```diff
@@ -133,3 +144,49 @@ exports.markersForArgs = function(starting, numberOfArgs) {
   }
   return result.join(", ");
 };
+
+function doNothing() {
+}
+
+function initTiming() {
+  if (!logQueryLimit) {
+    return doNothing;
+  }
+  let caller = getCallerPosition(2);
+  if (caller == "skip") {
```

Comment:
I'm a little bit confused by this getCallerPosition-related stuff, it seems a little dangerously clever.

Could we instead issue the timing calls from the files that _call into_ `db.js`, like `servershot.js`? That might be an opportunity to pass a label into `initTiming` to identify the query, which would make it directly greppable. Would also get us moving in the direction of adding timing for arbitrary other events.